code/modules/atmospherics/machinery/pipes/bridge_pipe.dm

```diff
@@ -0,0 +1,46 @@
+/obj/machinery/atmospherics/pipe/bridge_pipe
+	icon = 'icons/obj/atmospherics/pipes/bridge_pipe.dmi'
+	icon_state = "bridge_center"
+
+	name = "bridge pipe"
+	desc = "A one meter section of regular pipe used to connect pipenets over pipes."
+
+	dir = SOUTH
+	initialize_directions = NORTH | SOUTH
+	pipe_flags = PIPING_CARDINAL_AUTONORMALIZE | PIPING_BRIDGE
+	device_type = BINARY
+
+	construction_type = /obj/item/pipe/binary
+	pipe_state = "bridge_center"
+
+	var/static/list/mutable_appearance/center_cache = list()
+
+/obj/machinery/atmospherics/pipe/bridge_pipe/New()
```

Comment:
Not `Initialize`?

Reply:
Added a comment explaining why

Reply:
Deal with it in another way even if it ends up being that you update appearance after init, there's no reason nowadays to be using New() except on datums.

Reply:
moved to init next commit